Backend/README.md

```diff
@@ -0,0 +1,25 @@
+# iDO Backend
+
+## What is this?
+
+This is the node.js express server that will provide the backend API for our react native frontend application. It is connected to an [mLab](https://mlab.com/) mongodb databse that is hosted in AWS. 
+
+## Development
+
+### Setup
+
+Download [keepassX](https://www.keepassx.org/downloads) (You may have to allow for third-party applications in your settings)
+
+Open `secrets.kdbx` and follow the instructions in the file `mlab db ido-development` under the development group.
+
+
+### Run
+
+```bash
+$ npm install
```

Comment:
npm ci && npm start (keep it short!) 🚀

Reply:
I was going to recommend introducing `npm ci` when we integrate CI/CD but there's no harm to add it now.

Reply:
For reference: https://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable

Reply:
it's better for dependency management locally as well (works more like yarn and only installs from the lock file). npm install is flakey

Reply:
No more `rm -rf node_modules` - works for me.